### ISSUE TITLE
Java10 support

### ISSUE THREAD

**Issue by hendriks73** [osundblad/intellij-annotations-instrumenter-maven-plugin#26]
Tried to run this with Java 10 and got

```
java.lang.IllegalArgumentException
	at org.objectweb.asm.ClassReader.<init>(Unknown Source)
	at org.objectweb.asm.ClassReader.<init>(Unknown Source)
	at org.objectweb.asm.ClassReader.<init>(Unknown Source)
	at com.intellij.NotNullInstrumenter.instrumentClass(NotNullInstrumenter.java:80)
	at com.intellij.NotNullInstrumenter.instrumentFile(NotNullInstrumenter.java:72)

```
I suppose it might be caused by an unsupported class file version. But I'm just guessing.

**Comment by Trinova** [comment]
I have implemented a fix for it in #27 which I tested on our 22kLOC project. If someone else could verify that it works I'd appreciate it.

**Comment by osundblad** [comment]
In release 0.6.8
